scripts/xcode-test.sh

```diff
@@ -65,20 +54,54 @@ esac
 
 echo "CONFIGURATION: $CONFIGURATION"
 
-# The following tests fail on iOS 12.4. We ignore them for now and are going to fix them later.
-if [ $PLATFORM == "iOS" -a $OS == "12.4" ]; then
-    echo "Skipping tests for iOS 12.4."
+# There are some tests that are known to flake on Xcode 13.2.1 and 12.5.1 runs. They need to be handled separately due to new API introduced in Xcode 13.
+if [ $XCODE == "13.2.1" ]; then
```

Comment:
I wonder if we can match the start of the string, so if it's 13 do this, if it's 12 do that. Would be more future proof than these exact version matches. But my shell scripting sucks 😅

Reply:
Good call! done in https://github.com/getsentry/sentry-cocoa/pull/2089/commits/2aebecc7a9f23b6519c9977cc7d77f0dae45b989